pkg/initializer_v2/test/unit/model/test_model_config.py

```diff
@@ -0,0 +1,16 @@
+from pkg.initializer_v2.model.config import HuggingFaceModelInputConfig
+
+
+def test_huggingface_model_config_creation():
```

Comment:
Should we use @pytest.mark.parametrize with tests cases here for consistency across all unit tests ?